README.md

```diff
@@ -126,32 +115,58 @@ Workmanager.registerOneOffTask(
 ```
 
 ### BackoffPolicy
-When a job should fail this specifies the waiting strategy it should use.  
+Indicates the waiting strategy upon task failure.  
 The default is `BackoffPolicy.exponential`.    
-You can also specify the delay.  
+You can also specify the delay. 
 
 ```
 Workmanager.registerOneOffTask("1", "simpleTask", backoffPolicy: BackoffPolicy.exponential, backoffPolicyDelay: Duration(seconds: 10));
 ```
 
 ## Cancellation
 
-You can cancel jobs in different ways.  
-### By Tag
+A taks can be cancelled in different ways :  
+- ##### by Tag
 
-If you have provided job with a tag, you can cancel it that way too.  
+Cancels the task that was previously registered using this **Tag**, if any.  
 
 ```
 Workmanager.cancelByTag("tag");
 ```
 
-### By Unique Name
+- ##### by Unique Name
 ```
-Workmanager.cancelByUniqueName("tag");
+Workmanager.cancelByUniqueName("<MyTask>");
 ```
 
-### All
+- ##### cancel all registered tasks
 
 ```
 Workmanager.cancelAll();
 ```
+
+## iOS usage
+
+Background taks on iOS are very different. Before anything, make sure you've added the following key to your project's `Info.plist :
```

Comment:
Is this something that you typically do in XCode or just by hand?
Otherwise maybe a screenshot? I always place these in a `.art` folder in the root of the project

Reply:
This is left to the developer's taste ; Xcode won't compile the project if either the key or its value is incorrectly formatted anyway, and this step is very straightforward for anyone experienced with Xcode